docs/release-notes/2.27.0.md

```diff
@@ -0,0 +1,161 @@
+# Percona Monitoring and Management 2.27.0
+
+<table class="docutils field-list" frame="void" rules="none">
+  <colgroup>
+    <col class="field-name">
+    <col class="field-body">
+  </colgroup>
+  <tbody valign="top">
+    <tr class="field-odd field">
+    <th class="field-name">Date:</th>
+    <td class="field-body">April 14, 2022</td>
+    </tr>
+    <tr class="field-even field">
+      <th class="field-name">Installation:</th>
+      <td class="field-body">
+        <a class="reference external" href="https://www.percona.com/software/pmm/quickstart">Installing Percona Monitoring and Management</a></td>
+    </tr>
+  </tbody>
+</table>
+
+Percona Monitoring and Management (PMM) is a free and open-source platform for managing and monitoring MySQL, MongoDB, and PostgreSQL performance.
+
+!!! caution alert alert-warning "Important"
+    We recommend using the latest version of the PMM instance. This ensures that you access the latest features of the underlying components, such as VictoriaMetrics, with all the bug fixes in place. One of the critical features of VictoriaMetrics is [stream parsing mode](https://docs.victoriametrics.com/vmagent.html#stream-parsing-mode), which enhances the performance of PMM and saves memory when scraping targets expose millions of metrics.
+
+## Release Highlights
+
+### PMM and Percona Portal Integration
+
+- **Enhanced UX for connecting PMM server to Percona Portal**
+
+    To leverage Percona Platform's recent support for federated identity, PMM now uses access-token authorization for connecting PMM instances to Percona Platform. 
+
+    This replaces the former username/password authentication used in PMM 2.26.0 and older versions.
+
+    For more information, see [Integrate PMM with Percona Platform](https://www.percona.com/doc/percona-monitoring-and-management/2.x/how-to/integrate-platform.md). 
```

Comment:
is this correct link ? `integrate-platform.md` at least it doesn't work on rendered preview.

Reply:
@catalinaadam could you please check.

Reply:
@denisok, @rasika-chivate these are links to topics that got added after this branch was created. They are available with the Online Help changes  in https://github.com/percona/pmm-doc/pull/756

Reply:
or it should be a relative ling to internal `md` and that would be rendered and converted to `html` ?

Reply:
@denisok we normally provide HTML links in the Release Notes.

Reply:
@catalinaadam Could you please change that...

Reply:
these are new topics so I would have to guess the HMTLs. 
Is it that important to have html topics in just particular topic Rel Notes topics, whereas we have .md references in all other ones?

Reply:
@catalinaadam. It's not about guessing the HTML. I can help with the links.

Reply:
@denisok, you were right. Checked and fixed the links should be ok now